renderers/ebitengine/ebitengine.go

```diff
@@ -0,0 +1,484 @@
+package ebitengine
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+	"log/slog"
+	"math"
+	"strings"
+	"unsafe"
+
+	"github.com/hajimehoshi/ebiten/v2"
+	"github.com/hajimehoshi/ebiten/v2/text/v2"
+	"github.com/hajimehoshi/ebiten/v2/vector"
+	"github.com/totallygamerjet/clay"
+)
+
+var (
+	whiteImage *ebiten.Image
+)
+
+func init() {
+	img := ebiten.NewImage(3, 3)
+	img.Fill(color.White)
+	whiteImage = img.SubImage(image.Rect(1, 1, 2, 2)).(*ebiten.Image)
```

Comment:
Why create a SubImage?

Reply:
It is to prevent bleeding edges: https://github.com/hajimehoshi/ebiten/blob/1a4237213c92be1b9c16176887d992eb4183751b/vector/util.go#L26-L29
It's a habit, because I think it can happen during scaling/rotation operations, maybe not in this case though

Reply:
Ok let's leave a comment explaining why it's done like this

Reply:
Done!